azure-pipelines.yml

```diff
@@ -126,3 +133,15 @@ stages:
           build: $(Build.BuildNumber)
           agentPool: ${{ variables.agentPool }}
           terraformVersion: ${{ variables.terraformVersion }}
+
+  - stage: Prod_Apply
+    dependsOn: Aat_Apply
+    displayName: "Prod Apply"
+    condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
+    jobs:
+      - template: pipeline-templates/terraform-apply.yaml
+        parameters:
+          zone: 'prod'
```

Comment:
The zone isn’t prod isn’t prod the environment is

Reply:
Zone corresponds to all the zones in the prod environment. This is just to give the tfstate file a name. Since we are following the naming convention of segregating the zones based on which env they correspond to.